f8a_report/helpers/report_helper.py

```diff
@@ -140,7 +140,7 @@ def cleanup_db_tables(self):
                                'package_analyses', 'started_at', num_days)
 
         # Number of days to retain the package analyses data
-        num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '30')
+        num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '31')
```

Comment:
Any reason for this change?

Reply:
I looked at the logs of the job for the last 4 days and it passed for the last 2 days but failed for the days before that.From this i concluded that the time at which data is getting stored might be the reason.

Reply:
But how does changing it by one day help?

Reply:
As the task runs daily the maximum difference can be that one day.